packages/reactivity/src/ref.ts

```diff
@@ -492,6 +492,7 @@ export type UnwrapRef<T> = T extends ShallowRef<infer V>
 
 export type UnwrapRefSimple<T> = T extends
   | Function
+  | Date
```

Comment:
Considering adding more types, like `RegExp`

Reply:
Or can we use `Primitive` and `Builtin` here?

https://github.com/vuejs/core/blob/0e8bbe873e579f3d3a74c44af28f7df9e7a06978/packages/reactivity/src/reactive.ts#L143-L144

Reply:
Yes! Let's use `Builtin`

Reply:
Done.